### PR TITLE
Fix checkmark view

### PR DESCRIPTION
**Pull request by LePips**
Scales the background shape for the checkmark symbol such that it fills in the center transparent checkmark while staying inside the circle.

iOS 15 _looks like_ it will allow us to set the transparent checkmark white but I'm not entirely sure, we'll figure it out when we get there.

### CODE REVIEW

File: JellyfinPlayer/Components/PortraitItemView.swift [hunk 1]
```diff
@@ -47,7 +47,7 @@ struct PortraitItemView: View {
                                 Image(systemName: "checkmark.circle.fill")
                                     .foregroundColor(.accentColor)
                                     .background(Color(.white))
```
Comment (acvigue) [inline JellyfinPlayer/Components/PortraitItemView.swift:49]: If we're thinking in the mindset of setting background/foreground colors directly on the icon, this icon supports multicolor mode from iOS 14 & up (i.e. the intended targets)
Screen Shot 2021-07-31 at 7 50 24 AM

Comment (acvigue) [reply JellyfinPlayer/Components/PortraitItemView.swift:49]: I'd rather do this then continue with my hacky way of doing this :p

Comment (LePips) [reply JellyfinPlayer/Components/PortraitItemView.swift:49]: The multicolor support in iOS 14 **only** allows for it to be that blue color with the white checkmark.

In iOS 15, we can change the palette of the colors for the symbol but it is only supported in iOS 15.

Look at section: _How to create a custom SF Symbols palette_
https://www.hackingwithswift.com/articles/237/complete-guide-to-sf-symbols

Comment (LePips) [reply JellyfinPlayer/Components/PortraitItemView.swift:49]: The hack you did was good and it's necessary here

Comment (acvigue) [reply JellyfinPlayer/Components/PortraitItemView.swift:49]: > The multicolor support in iOS 14 _only_ allows for it to be that blue color with the white checkmark.
> 
> In iOS 15, we can change the palette of the colors for the symbol but it is only supported in iOS 15.
> 
> Look at section: _How to create a custom SF Symbols palette_
> https://www.hackingwithswift.com/articles/237/complete-guide-to-sf-symbols

Oh, didn't know that. That wouldn't work as we use the purple accent color.
this lgtm!

**Review by acvigue — APPROVED** [review]

**Pull request merged**